<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmIpcPolling</class>
 <widget class="QWidget" name="frmIpcPolling">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <property name="spacing">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="labAutoPoll">
        <property name="text">
         <string>自动轮询</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="SwitchButton" name="btnAutoPoll" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labPollInterval">
        <property name="text">
         <string>轮询间隔</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboxPollInterval">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labPollType">
        <property name="text">
         <string>轮询画面</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="cboxPollType"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="labPollRtsp">
        <property name="text">
         <string>码流类型</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="cboxPollRtsp"/>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QListWidget" name="listWidget"/>
      </item>
      <item row="5" column="0" colspan="2">
       <widget class="QLabel" name="labTip">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>25</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string>第 0 次 / 第 0 组 / 共 0 组</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="btnPollStart">
          <property name="text">
           <string>启动轮询</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPollPause">
          <property name="text">
           <string>暂停轮询</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
